APItokenmanager/Web/migrations/0001_initial.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.18 on 2024-04-25 13:12
```

Comment:
Since this is your initial merge, you should be able to merge migrations here -- until each PR is additive, one migration to start off your deployment would be much more efficient